apis/core/lib/domain/cube-projects/import.ts

```diff
@@ -103,7 +101,7 @@ export async function importProject({
   const projectNode = await store.createMember(projectsCollection.term, id.cubeProject(label))
 
   const project = createMinimalProject(projectNode, {
-    creator: { id: user, name: userName },
+    creator: user,
```

Comment:
Since we now have graphs for users, only the URI will be stored with the project now, unlike the lazy duplication we've had so far